specifyweb/specify/models.py

```diff
@@ -1466,6 +1466,7 @@ class Collectionobject(models.Model):
     paleocontext = models.ForeignKey('PaleoContext', db_column='PaleoContextID', related_name='collectionobjects', null=True, on_delete=protect_with_blockers)
     visibilitysetby = models.ForeignKey('SpecifyUser', db_column='VisibilitySetByID', related_name='+', null=True, on_delete=protect_with_blockers)
     collectionobjecttype = models.ForeignKey('CollectionObjectType', db_column='CollectionObjectTypeID', related_name='collectionobjects', null=True, on_delete=models.SET_NULL)
+    coparentcog = models.ForeignKey('CollectionObjectGroup', db_column='CoParentCogID', related_name='+', null=True, on_delete=protect_with_blockers)
```

Comment:
(optional) Would it make more sense to name the field `parentCog` so the name is consistent across both COG and CO tables?